components/automate-ui/src/app/modules/license/license-lockout/license-lockout.component.ts

```diff
@@ -123,12 +120,12 @@ export class LicenseLockoutComponent implements AfterViewInit {
 
     this.clearErrors();
     this.requestingLicense = true; // round-trip initiating...
-    this.store.dispatch(new RequestLicense(trialLicenseRequest));
+    this.licenseFacade.dispatch(new RequestLicense(trialLicenseRequest));
```

Comment:
Similar to above, suggest move dispatches into facade
(similar to components/automate-ui/src/app/entities/projects/project.service.ts)